analysis_options.yaml

```diff
@@ -14,6 +14,8 @@ analyzer:
     # necessary
     camel_case_types: ignore
     non_constant_identifier_names: ignore
+    # Consider removing from dart_flutter_team_lints.
+    unreachable_from_main: ignore
```

Comment:
I like this lint. How'd you hit it?

Reply:
In the `test_fixes/renames.dart` file (which to be honest isn't really intended to be best practice dart code). From memory, I've also hit it a few other times where it was false positives.